### PR TITLE
Feature/games table

### PR DESCRIPTION
**Pull request by vinayak-4**

### CODE REVIEW

File: src/hooks/useGameData.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import { useQuery } from 'react-query';
+
+function fetchGames() {
+  return axios.get('http://localhost:1337/api/games');
+}
+
+export default function useGameData() {
```
Comment (Delwalt) [inline src/hooks/useGameData.js]: do you really need this extra hook, isn't `useQuery` enough ?

File: src/components/Games/Index.jsx [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+    title: 'Released',
+    dataIndex: 'Released',
+    key: 'released',
+  },
+];
+
 export default function Games() {
-  return <div>Games</div>;
+  const { data } = useGameData();
+  const tableData = data?.map(({ id, attributes }) => {
```
Comment (Delwalt) [inline src/components/Games/Index.jsx:65]: what about loading, error state ?

**Review by Delwalt — CHANGES_REQUESTED** [review]

File: src/components/Games/Index.jsx (hunk 2 above)
Comment (vinayak-4) [reply src/components/Games/Index.jsx:65]: Sorry sir

Comment (vinayak-4) [reply src/components/Games/Index.jsx:65]: I have fixed it now

**Review by Delwalt — APPROVED** [review]

**Pull request merged**